kitsune/users/static/users/js/actions/UserAuthActions.es6.js

```diff
@@ -55,9 +55,15 @@ export function login(username, password, {inactive=false}={}) {
     }
 
     if (newDoc.find('#login').length === 0) {
-      Dispatcher.dispatch({
-        type: actionTypes.AUTH_LOG_IN_SUCCESS,
-        username,
+      return apiFetch(`/api/2/user/${username}/`, {
+        method: 'GET',
+      })
+      .then(result => {
+        Dispatcher.dispatch({
+          type: actionTypes.AUTH_LOG_IN_SUCCESS,
+          id: result.id,
```

Comment:
The function `checkAuthState` in this file will also trigger a `AUTH_LOG_IN_SUCCESS` action if the user comes to the page already signed in. You need to update the ID there too.